shardingsphere-governance/shardingsphere-governance-context/src/main/java/org/apache/shardingsphere/governance/context/process/GovernanceExecuteProcessReporter.java

```diff
@@ -29,12 +35,14 @@
 public final class GovernanceExecuteProcessReporter implements ExecuteProcessReporter {
     
     @Override
-    public void report(final SQLStatementContext<?> context, final ExecutionGroupContext<? extends SQLExecutionUnit> executionGroupContext, final ExecuteProcessConstants constants) {
-        // TODO :Call API of configCenter
+    public void report(final SQLStatementContext<?> context, final ExecutionGroupContext<? extends SQLExecutionUnit> executionGroupContext) {
+        ExecuteProcessContext executeProcessContext = new ExecuteProcessContext(executionGroupContext);
+        ShardingSphereEventBus.getInstance().post(new ReportExecuteProcessSummaryEvent(new YamlExecuteProcessContext(executeProcessContext)));
```

Comment:
Why is a new class, i.e.,`YamlExecuteProcessContext` used here? `ExecutionGroupContext` is unsuitable?

Reply:
It is suggested to consider renaming `ReportExecuteProcessUnitEvent` with `ExecuteProcessUnitReportEvent `. Similarly, `ReportExecuteProcessSummaryEvent` -> `ExecuteProcessSummaryReportEvent `.